app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        new_file = command.split()
```

Comment:
You are splitting the command string again. Instead, store the result of `command.split()` in a variable and reuse it to improve efficiency.